system/services/helpers/JQueryDatatablesHelpers.cfc

```diff
@@ -29,6 +29,9 @@ component {
 			ArrayAppend( result.aaData, record );
 		}
 
+		if( isStruct( result.aaData[1] ) AND result.aaData[1]._options NEQ '' )
```

Comment:
I'm not sure about this. Firstly:

* This presumably should be run against each row
* The cleanup needs to be a little smarter than just removing all whitespace?

I imagine the whitespace cleanup would be something like:

```reReplace( result.aaData[ i ]._options, "\s+", " ", "all" )

(i.e. tidyup double spaces, not get rid of all).

Would be good to see some before and after output after making this fix.